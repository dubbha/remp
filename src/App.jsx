import React from 'react';

const App = () => <div>React Mentoring Program</div>;

export default App;
