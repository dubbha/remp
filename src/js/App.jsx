import React from 'react';
import Footer from './footer';
import List from './list';
import Search from './search';
import Film from './film';

const App = () => (
  <div className="app">
    { true && <Search /> }
    { false && <Film /> }
    <List />
    <Footer />
  </div>
);

export default App;
