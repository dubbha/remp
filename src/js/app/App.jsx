import React from 'react';
import { Switch } from 'react-router-dom';
import { renderRoutes } from 'react-router-config';
import routes from './routes';
import './style.sass';

const App = () => (
  <div className="app">
    <Switch>
      {renderRoutes(routes)}
    </Switch>
  </div>
);

export default App;
