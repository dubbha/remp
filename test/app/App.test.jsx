import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import App from 'app/App.jsx';

describe('app/App', () => {
  it('should render successfully', () => {
    const wrapper = shallow(<App />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
