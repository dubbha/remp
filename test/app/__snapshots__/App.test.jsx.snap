// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app/App should render successfully 1`] = `
<div
  className="app"
>
  <Switch>
    <Switch>
      <Route
        key="0"
        path="/search/:query?"
        render={[Function]}
      />
      <Route
        key="1"
        path="/film/:id/:title?"
        render={[Function]}
      />
      <Route
        exact={true}
        key="2"
        path="/"
        render={[Function]}
      />
    </Switch>
  </Switch>
</div>
`;
